src/ORM/Connect/DBSchemaManager.php

```diff
@@ -436,11 +434,10 @@ public function requireTable(
             if (!$table_name_info_sent) {
                 $this->alterationMessage(
                     <<<'MESSAGE'
-<strong>Please note:</strong> It is strongly recommended to define a
-table_name for all namespaced models. Not defining a table_name may cause generated table
-names to be too long and may not be supported by your current database engine. The generated
-naming scheme will also change when upgrading to SilverStripe 5.0 and potentially break.
-MESSAGE
+                    <strong>Please note:</strong> It is strongly recommended to define a
+                    table_name for all namespaced models. Not defining a table_name may cause generated table
+                    names to be too long and may not be supported by your current database engine.
+                    MESSAGE
```

Comment:
With the alignment, the extra spaces are part of the text.

```suggestion
MESSAGE
```

Reply:
That shouldn't be the case as of PHP 7.3: https://www.php.net/manual/en/migration73.new-features.php#migration73.new-features.core.heredoc

I also don't see that indentation being output when I test this locally - neither in the rendered output nor directly in the markup:
![rendered output](https://user-images.githubusercontent.com/36352093/186019467-6d4f9de4-532a-4e5e-84fa-50d30c1524df.png)
```html
<li class="success">Table App_ExampleClass1: created</li><li class="success">Field App_ExampleClass1.ID: created as int(11) not null auto_increment</li><li class="success">Field App_ExampleClass1.ClassName: created as enum('App\\One\\Two\\Three\\ExampleClass1') character set utf8mb4 collate utf8mb4_unicode_ci default 'App\\One\\Two\\Three\\ExampleClass1'</li><li class="success">Field App_ExampleClass1.LastEdited: created as datetime</li><li class="success">Field App_ExampleClass1.Created: created as datetime</li><li class="success">Index App_ExampleClass1.ClassName: created as index ("ClassName")</li><li class="error"><strong>Please note:</strong> It is strongly recommended to define a
table_name for all namespaced models. Not defining a table_name may cause generated table
names to be too long and may not be supported by your current database engine.</li><li class="warning">table_name not set for class App\One\Two\Three\ExampleClass1</li>
```

Were you actually seeing the extra spaces being part of the text when trying this locally?

Reply:
It's a slight improvement in formatting, though at the same time please refrain from refactoring/reformat stuff in PR's that's unrelated to the original task of rescuing an old commit.  The just adds unnecessary confusion and communication (3x devs discussing this tiny detail) which greatly slows down the peer review process and isn't worth the tiny improvement in code quality